### DISCUSSION TITLE
现在是不是没有首页卡片布局功能啊

### DISCUSSION THREAD

**Discussion by veizei-K** [the1812/Bilibili-Evolved#4661] [功能建议 / Ideas]
![image](https://github.com/the1812/Bilibili-Evolved/assets/129288685/4ebe2c06-3f80-4c22-a9af-a1ec53f274b2)
比如像这样。

**Comment by Geeyun-JY3** [comment]
破站现在最新版首页不就是这样的吗？

**Reply by veizei-K** [reply]
是，但是开了缩放之后就没办法设定列数了。

**Comment by the1812** [comment]
没有

**Comment by Geeyun-JY3** [comment]
今天逛 Greasy Fork 偶然发现个脚本好像有相关功能，或者可以试试：[哔哩哔哩新版首页排版调整和去广告(bilibili)](https://greasyfork.org/scripts/477487)
注：我没实际安装测试过是否兼容，不知道实际效果如何，只是看其简介上写有相关功能。

**Reply by veizei-K** [reply]
我在用bilibili页面净化大师，也挺好用
